hack/Jenkins/W2L/setup.sh

```diff
@@ -199,9 +199,9 @@ if [ $ec -eq 0 ]; then
 	export TIMEOUT="5m"
 	export DOCKER_HOST="tcp://$ip:$port_inner"
 	export DOCKER_TEST_HOST="tcp://$ip:$port_inner"
-	unset DOCKER_CLIENTONLY
+    unset DOCKER_CLIENTONLY
```

Comment:
weird indentation, no big deal, just in case you repush the PR, might as well fix it :)